src/api/v1/organisations/id.ts

```diff
@@ -0,0 +1,25 @@
+import * as Hapi from 'hapi';
+import { getCommunityBusiness } from '../prerequisites';
+import { query, response } from './schema';
+
+
+export default [
+  {
+    method: 'GET',
+    path: '/organisations/{organisationId}',
+    options: {
+      description: 'Retreive list of all organisations',
+      auth: {
+        strategy: 'standard',
+        access: {
+          scope: ['organisations_details-child:read', 'organisations_details-own:read'],
+        },
+      },
+      validate: { query },
+      response: { schema: response },
+      pre: [{ method: getCommunityBusiness, assign: 'communityBusiness' }],
```

Comment:
This is mixing up CBs and orgs. Take a look at #118, I added `GET /community-businesses/:id`, maybe you can cherry pick something useful

Reply:
Yeah this route is effectively for cbs rather than organisations, I'll rewrite it to the latter. 

Can you please add the routes your working on in #83. I've been asking for a month now.